tests/ui/test_non_admin_user.py

```diff
@@ -3,8 +3,17 @@
 
 
 from ocs_ci.ocs.ui.mcg_ui import ObcUi
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
 from ocs_ci.ocs import ocp
-from ocs_ci.framework.testlib import ManageTest, ui, bugzilla, polarion_id, tier2
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    ui,
+    bugzilla,
+    polarion_id,
+    tier2,
```

Comment:
Now it shows both tier 1 and 2

Reply:
@am-agrawa tier2 is being used by other test.